src/main/java/com/gromoks/filestatistic/service/statistic/FileStatistic.java

```diff
@@ -0,0 +1,124 @@
+package com.gromoks.filestatistic.service.statistic;
+
+import com.gromoks.filestatistic.entity.FileInfo;
+import com.gromoks.filestatistic.entity.LineInfo;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class FileStatistic {
+    public static List<LineInfo> getLinesStatistic(InputStream lines, String wordDelimiter) {
+        List<LineInfo> lineInfos = new ArrayList<>();
+
+        try (BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(lines))) {
+            String line;
+            int lineNumber = 0;
+
+            while ((line = bufferedReader.readLine()) != null) {
+                lineNumber++;
+                LineInfo lineInfo = FileStatistic.getLineStatistic(line, lineNumber, wordDelimiter);
```

Comment:
Remove (new FileStatistic) instatiation from the method.
Committed code version does not contain this part, but you know what I'm talking about :-)